crates/core/machine/src/utils/prove.rs

```diff
@@ -674,12 +677,19 @@ where
         // Print the opcode and syscall count tables like `du`: sorted by count (descending) and
         // with the count in the first column.
         tracing::info!("execution report (opcode counts):");
-        for line in sorted_table_lines(report_aggregate.opcode_counts.as_ref()) {
-            tracing::info!("  {line}");
+        let (width, lines) = sorted_table_lines(report_aggregate.opcode_counts.as_ref());
```

Comment:
why is this logic duplicated from the executor? i notice it was before your changes as well -- just wondering

Reply:
Indeed I see no reason, I'll see to refactor

Reply:
Mmm, In report.rs we have writeln!, in prove.rs it's tracing::info!